hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/store/driver/TestStateStoreDriverBase.java

```diff
@@ -574,6 +578,15 @@ private static Map<String, Class<?>> getFields(BaseRecord record) {
     return getters;
   }
 
+  public void testCacheLoadMetrics(StateStoreDriver driver, int numRefresh)
+      throws IOException, IllegalArgumentException {
+    StateStoreMetrics metrics = stateStore.getMetrics();
+    final Query<MountTable> query = new Query<>(MountTable.newInstance());
+    driver.getMultiple(MountTable.class, query);
+    assertEquals(numRefresh,
```

Comment:
Can we check for NumOps and Time?

Reply:
The only problem with time duration check is that it could result in flaky tests in future. Is numOps alone fine?

Reply:
Done, added time as well. I have added time duration value high enough that it's highly unlikely that it could fail to assert the value.